### PR TITLE
Created chevereto proxy sample

### PR DESCRIPTION
**Pull request by rastacalavera**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]
	

made this since this image is offered by LSIO and one currently does not exist.

### CODE REVIEW

File: chevereto.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## Version 2020/12/09
```
Comment (aptalca) [inline chevereto.subdomain.conf.sample:1]: These two lines seem to be forgotten

**Review by rastacalavera — COMMENTED** [review]
woops must have deleted that, thanks

File: chevereto.subdomain.conf.sample [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+        # enable the next two lines for ldap auth
+        #auth_request /auth;
+        #error_page 401 =200 /ldaplogin;
+
+        # enable for Authelia
+        #include /config/nginx/authelia-location.conf;
+
+        include /config/nginx/proxy.conf;
+        resolver 127.0.0.11 valid=30s;
```
Comment (Roxedus) [inline chevereto.subdomain.conf.sample:33]: Please update this to reflect the changes brought by #330

**Review by Roxedus — CHANGES_REQUESTED** [review]

**Comment by drizuid** [comment]
closing as https://github.com/linuxserver/reverse-proxy-confs/pull/350 was merged

**Pull request closed without merging**